<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StrangerPrefsWindow</class>
 <widget class="QDialog" name="StrangerPrefsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Search for partners who speak</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="languagePicker">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Search for partners with interests</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="InterestsTextEdit" name="interestsTextbox">
     <property name="maximumBlockCount">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InterestsTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>intereststextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>languagePicker</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>StrangerPrefsWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>StrangerPrefsWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
